setup.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
Likewise, remove this and revert the mode to 0644?

The recommended way to use `setup.py` in `acmens` is via `make`. If a setup command is missing in the Makefile, we can add it as a phony target (preferably in a separate PR).